README.md

```diff
@@ -23,7 +23,7 @@ cd resetti
 makepkg -si
 ```
 
-### From the Debian package
+### From the Debian or RPM package
 
 Check the [Releases](https://github.com/woofdoggo/resetti/releases) tab for
 Debian packages or download the latest development builds from the
```

Comment:
Since it's no longer Debian packages only, the text underneath the header could be adjusted to say e.g. "distro-specific packages" instead of Debian packages. The header should probably be changed to be generic if and when support for a 3rd packaging format gets added, but it's fine for now.